# Lab5 Ballistic Pendulum Calculations

Method 2

In [13]:
import numpy as np
#vi=x(g^1/2)(2y)^1/2
def rule_4(Q,m,n,dA,A,dB,B):
    dQ=Q*(np.sqrt(((((m*dA)/(A))**2))+(2*((n*dB)/(B))**2)))
    return dQ
#x:A,m
#y:B,n
Q=4.75
m=1
n=-0.5
dA=0.0005
A=2.1369
dB=7.07e-4
B=0.990

dx=rule_4(Q,m,n,dA,A,dB,B)
print ('error=',dx)


error= 0.002643610365260536


The error calcuation from the lab = $3.57X10^-3$ m/s. This is not the same as the python error. I am supposing because of a calcualtion error somewhere

# Python Calcs

In [17]:
x=np.array([1.1, 1.3, 1.4, 0.9, 0.95, 1.05])
avg=np.average(x)
print('avg=',avg)

sd=np.std(x)
err=sd/(np.sqrt(6))
print('error=',err)

avg= 1.1166666666666667
error= 0.073282810879294


The average of the 6 values is 1.1 and the error is 0.073.